src/server/search/search_family_test.cc

```diff
@@ -428,4 +428,32 @@ TEST_F(SearchFamilyTest, FtProfile) {
   }
 }
 
+TEST_F(SearchFamilyTest, BasicSort) {
+  Run({"ft.create", "i1", "schema", "ord", "numeric", "sortable"});
+
+  for (size_t i = 0; i < 100; i++)
+    Run({"hset", absl::StrCat("d:", i), "ord", absl::StrCat(i)});
+
+  auto AreRange = [](size_t total, size_t l, size_t r) {
+    vector<string> out;
+    for (size_t i = min(l, r); i < max(l, r); i++)
+      out.push_back(absl::StrCat("d:", i));
+    if (l > r)
+      reverse(out.begin(), out.end());
+    return DocIds(total, out);
+  };
+
+  // Sort ranges of 23 elements
+  for (size_t i = 0; i < 77; i++)
+    EXPECT_THAT(
+        Run({"ft.search", "i1", "*", "SORTBY", "ord", "LIMIT", to_string(i), to_string(23)}),
+        AreRange(100, i, i + 23));
+
+  // Sort ranges of 27 elements in reverse
+  for (size_t i = 0; i < 73; i++)
+    EXPECT_THAT(Run({"ft.search", "i1", "*", "SORTBY", "ord", "DESC", "LIMIT", to_string(i),
```

Comment:
I guess it's working well, just the comment line then :)

Reply:
Didn't understand 🙂